oat-sa/tao-core
# Fix/ms 614 content container width

Related to : https://oat-sa.atlassian.net/browse/MS-614
 
Fix content area header width style when navbar is hidden
  
#### How to test

- navigate to backoffice
- open page without navbar (for example, extensions list
- ensure content area header has a full width

#### Related PR's
none
